### PR TITLE
Add Insight anonymously report usage statistics

### PR DESCRIPTION
**Pull request by zckrs**

### CODE REVIEW

**Comment by Swiip** [comment]
:+1: 

Still WIP ?

**Comment by zckrs** [comment]
Still WIP. Actualy we just track the first run generator.

**Comment by zckrs** [comment]
Options:

* track each prompt selection in different event ('/jquery2', '/restangular', '/ui.router', ...)
* track user profil ('/jquery2/restangular/ui.router/...', '/zeptoJS/$http/new-router/...', ...)
* or both

**Comment by zckrs** [comment]
Examples data send by Insight:

* `/jquery2/angular-resource/ui-router/bootstrap/noBootstrapComponents/node-sass/noJsPrepro/noHtmlPrepro`
* `/jqLite/restangular/new-router/material-design-lite/null/less/typescript/jade`

File: app/src/prompts.js [hunk 1]
```diff
@@ -137,4 +151,12 @@ module.exports = function(GulpAngularGenerator) {
     }.bind(this));
   };
 
+  /**
+   * Send anonymously report usage statistics by Insight
+   */
+  GulpAngularGenerator.prototype.sendInsight = function sendInsight() {
+
+    this.insight.track(this.props.jQuery.key, this.props.resource.key, this.props.router.key, this.props.ui.key, this.props.bootstrapComponents.key, this.props.cssPreprocessor.key, this.props.jsPreprocessor.key, this.props.htmlPreprocessor.key);
```
Comment (zckrs) [inline app/src/prompts.js]: Ugly line.
@Swiip should I use lodash to compose a object ?

Comment (Swiip) [inline app/src/prompts.js]: Yep, perhaps it's better. lodash -> array -> this.insight.track.apply

**Pull request merged**